### DISCUSSION TITLE
typos: Multiple PR/MR or batch?

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.NET#1515] [General] [answered]
:wave: 

Sometimes it's just a typo.
Sometimes it's "like a typo", but it changes the meaning (like #1513... I should have name it differently).

For real "typos", would you prefer a:
- PR/MR / fix
- or a batch of multiple fix?

**Comment by dlemstra** [comment, marked as answer]
Feel free to do what you prefer for those typos pr's. For typo's that are coding mistakes I really want a separate PR with a fix.
